docs/niscope/session.rst

```diff
@@ -108,13 +108,13 @@ niscope.Session
    +-----------------------------------------------------+---------------------------------------+
    | :py:attr:`fetch_meas_num_samples`                   | int                                   |
    +-----------------------------------------------------+---------------------------------------+
-   | :py:attr:`fetch_num_records`                        | int                                   |
+   | :py:attr:`_fetch_num_records`                       | int                                   |
```

Comment:
Private attributes are still in the RST, they should be removed.

Reply:
Removed private attributes.